it/test/connectors/IdentityMatchConnectorSpec.scala

```diff
@@ -28,98 +29,104 @@ import play.api.http.Status._
 import play.api.inject.guice.GuiceApplicationBuilder
 import play.api.test.DefaultAwaitTimeout
 import play.api.test.Helpers.CONTENT_TYPE
-import suite.{BaseSuite, WireMockHelper}
+import suite.BaseSuite
 import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.http.test.WireMockSupport
 
-import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
 
 class IdentityMatchConnectorSpec extends AnyWordSpec with BaseSuite
   with Matchers
   with GuiceOneAppPerSuite
   with ScalaFutures
   with DefaultAwaitTimeout
-  with WireMockHelper
+  with WireMockSupport
   with IntegrationPatience
   with EitherValues {
 
   private def applicationBuilder(): GuiceApplicationBuilder = new GuiceApplicationBuilder()
     .configure(
-      "microservice.services.individual-match.port" -> server.port(),
+      "microservice.services.individual-match.port" -> wireMockServer.port(),
       "mongodb.uri" -> "mongodb://localhost:27017/individual-check-it",
       "metrics.enabled" -> false,
       "auditing.enabled" -> false
     )
 
   private lazy val application = applicationBuilder().build()
 
-  implicit val headerCarrier: HeaderCarrier = mock[HeaderCarrier]
+  implicit lazy val headerCarrier: HeaderCarrier = HeaderCarrier()
 
   private def identityMatchConnector = application.injector.instanceOf[IdentityMatchConnector]
 
+  private def getMatchIdResponse(request: IdMatchRequest, connector: IdentityMatchConnector): Future[IdMatchApiResponse] = {
+    connector.matchId(
+      request.nino,
+      request.surname,
+      request.forename,
+      request.birthDate
+    )
+  }
+
   "Identity Match Connector" should {
 
     "parse response correctly" when {
 
       "successful response is returned from the API" in {
 
-        server.stubFor(post(urlEqualTo("/individuals/match"))
+        wireMockServer.stubFor(post(urlEqualTo("/individuals/match"))
```

Comment:
This can be put in a method in the BaseSuite as well.

Reply:
Added method